core/src/main/java/com/linecorp/armeria/client/AbstractRuleBuilder.java

```diff
@@ -160,32 +151,39 @@ public AbstractRetryRuleBuilder onException(Predicate<? super Throwable> excepti
     }
 
     /**
-     * Makes a {@link RetryRule} retry on any {@link Exception}.
+     * Adds any {@link Exception}.
      */
-    public AbstractRetryRuleBuilder onException() {
+    public AbstractRuleBuilder onException() {
         return onException(unused -> true);
     }
 
     /**
-     * Makes a {@link RetryRule} retry on an {@link UnprocessedRequestException} which means that the request
-     * has not been processed by the server. Therefore, you can safely retry the request without worrying about
-     * the idempotency of the request.
+     * Adds an {@link UnprocessedRequestException}.
      */
-    public AbstractRetryRuleBuilder onUnprocessed() {
+    public AbstractRuleBuilder onUnprocessed() {
         return onException(UnprocessedRequestException.class);
     }
 
-    Predicate<RequestHeaders> requestHeadersFilter() {
+    /**
+     * Returns the {@link Predicate} of a {@link RequestHeaders}.
+     */
+    public final Predicate<RequestHeaders> requestHeadersFilter() {
```

Comment:
protected?

Reply:
These getter are used in `internal.client.AbstractRuleBuilderUtil`. 😅

Reply:
Let me refactor it.